src/fu-daemon.c

```diff
@@ -2110,11 +2110,9 @@ fu_daemon_set_machine_kind(FuDaemon *self, FuDaemonMachineKind machine_kind)
 }
 
 gboolean
-fu_daemon_setup(FuDaemon *self, GError **error)
+fu_daemon_setup(FuDaemon *self, const gchar *socket_address, GError **error)
 {
 	const gchar *machine_kind = g_getenv("FWUPD_MACHINE_KIND");
-	const gchar *socket_filename = g_getenv("FWUPD_DBUS_SOCKET");
```

Comment:
I think this environment variable should be dropped in matching readme.

Also the client should probably use the same define.

Reply:
I think we still want it in Linux for the socket self tests, no?

Reply:
Oh I guess still want to use unix socket in Linux, right ok.